src/js/pages/Index.js

```diff
@@ -140,14 +142,58 @@ var Index = React.createClass({
     );
   },
 
+  handleWindowResize(event) {
+    // TODO: trottle method using RXJS
```

Comment:
Is there a jira to do so? I would love to have it linked, otherwise, we will forget and this comment is going to get stale pretty soon.

Reply:
No Jira yet, but I will create one or I might make it part of this PR.

Reply:
@weblancaster Would you mind adding the Jira for these todos still or removing them?